### DISCUSSION TITLE
How to use last engine SNAPSHOT version

### DISCUSSION THREAD

**Discussion by icrc-fdeniger** [google/android-fhir#2502] [Q&A] [answered]
Hello,

the last published version for engine is1.0.0 (built on 2024-02-07 is https://maven.google.com/web/index.html?q=fhir#com.google.android.fhir:engine:1.0.0.)
I'd like to use the very last SNAHPSHOT version.
How could I do that ?
I had a look to the build and didn't see an update to a public repo.

Thanks :)

**Comment by vorburger** [comment, marked as answer]
see https://google.github.io/android-fhir/use/Snapshots/ - Thank You @icrc-fdeniger for your contribution (#2525) for this!
